packages/components/src/internal/components/labels/BarTenderSettingsForm.tsx

```diff
@@ -101,173 +92,149 @@ const SettingsInput: FC<SettingsInputProps> = memo(({ children, description, lab
     );
 });
 
-// exported for jest testing
-export class BarTenderSettingsFormImpl extends PureComponent<Props, State> {
-    static defaultProps = {
-        api: getDefaultAPIWrapper(),
-    };
-
-    private btTestConnectionTemplate = (label: string): string => {
-        // Should be able to run connection test w/o a default label set.
-        const formatNode = `<Format>${label}</Format>`;
+const btTestConnectionTemplate = (label: string): string => {
+    // Should be able to run connection test w/o a default label set.
+    const formatNode = `<Format>${label}</Format>`;
 
-        return `<XMLScript Version="2.0">
+    return `<XMLScript Version="2.0">
             <Command Name="Job1">
                 <FormatSetup>
                     ${formatNode}
                 </FormatSetup>
             </Command>
         </XMLScript>`;
-    };
-
-    constructor(props: Props) {
-        super(props);
-
-        this.state = {
-            btServiceURL: undefined,
-            dirty: false,
-            submitting: false,
-            testing: false,
-            saveErrorMsg: undefined,
-            connectionValidated: undefined, // Don't know if it has been verified
-            failureMessage: undefined,
-        };
-    }
-
-    componentDidMount(): void {
-        this.load();
-    }
-
-    load = (): void => {
-        this.props.api.labelprinting.fetchBarTenderConfiguration().then(btConfiguration => {
-            this.setState(() => ({
-                btServiceURL: btConfiguration.serviceURL,
-                dirty: false,
-                submitting: false,
-            }));
+};
+const apiWrapper = getDefaultAPIWrapper();
```

Comment:
curious why this was moved to a const outside of the component instead of within the props deconstruction within?

Reply:
Because if the API isn't set and the default was used, we create a new object, this triggered anything that was dependent on the API object (or the labelprinting property).